src/app/sidebar/sidebar.component.html

```diff
@@ -18,7 +18,7 @@
             <i class="menu-icon fas fa-wallet"></i>
             <span>Wallet <i *ngIf="!hasWalletActive" class="ml-2 fas fa-exclamation-circle" style="color: orange;"></i></span>
         </li>
-        <li *ngIf="hasWalletActive" routerLink="manage_groups" routerLinkActive="side-active">
+        <li *ngIf="hasWalletActive" (click)="closeSideBar()" routerLink="manage_groups" routerLinkActive="side-active">
```

Comment:
Adding a click listener to this item is not needed. 
If you remind, there was an issue where, for example, when you open `Wallet` item in the sidebar, it will collapse, but when you click `Investment groups`, it will not collapse. The feature works, but not for items like `Investment groups`. The only difference between these items is using `*ngIf` for `li` items. The problem is because we are using jQuery which registers `li` items when the page loads. But damn jQuery is not aware that Angular manipulates with DOM. When you write `*ngIf`, if the evaluation of its expression returns `false` it will not be added to DOM, and jQuery won't be registered to it. However, once it evaluates to `true`, it will be added to DOM, buy jQuery won't listen to this item. This is the reason why we should avoid using jQuery. Hope you'll understand the reasons.

To avoid it, remove all `closeSideBar()` click listeners and just replace `*ngIf` with `*ngShow`. These will remain in DOM, but will be invisible.

Reply:
Thank you for that detailed description. Will try that way! (y)

Reply:
I've just tried out. just instead of:

```typescript
*ngIf="hasWalletActive"
```

set: 

```typescript
[hidden]="!hasWalletActive"
```